chain/core/tests.py

```diff
@@ -170,6 +178,13 @@ def setUp(self):
         self.sensors = []
         for device in self.devices:
             device.save()
+            self.metadata.append(Metadata(value="Test Metadata 1",
```

Comment:
I think the key is stored as an empty string because of the default not null constraint. We can use the ```MinLengthValidator``` if we want to make sure that key is not empty.